mac

```diff
@@ -106,6 +106,9 @@ config_zsh() {
 
     zsh_config_homebrew
 
+    # shellcheck disable=SC2016
+    append_to_zshrc 'export PATH=~/.asdf/shims:$PATH'
```

Comment:
If we added it here, should we remove it from `install_asdf()`?

Reply:
I don't think so because if the user decides to not install `oh-my-zsh` at the last step they won't have it at all.